docs/src/theme/DocCategoryGeneratedIndexPage/index.js

```diff
@@ -1,49 +0,0 @@
-/**
```

Comment:
Did you delete it on purpose?

Reply:
Yes! It was @carolinamenezes who deleted it!